src/WireMock.Net/Util/BodyParser.cs

```diff
@@ -107,7 +107,7 @@ public static BodyType DetectBodyTypeFromContentType([CanBeNull] string contentT
             return BodyType.Bytes;
         }
 
-        public static async Task<BodyData> Parse([NotNull] Stream stream, [CanBeNull] string contentType)
+        public static async Task<BodyData> Parse([NotNull] Stream stream, [CanBeNull] string contentType, bool deserializeJson)

```

Comment:
Maybe better to change the order from this parameter because contentType is optional?
Like
`Parse([NotNull] Stream stream, bool deserializeJson, [CanBeNull] string contentType)`

or even better , set it default to true:
`Parse([NotNull] Stream stream, bool deserializeJson = true, [CanBeNull] string contentType = null)`

Reply:
I changed it to 
`Parse([NotNull] Stream stream, [CanBeNull] string contentType = null, bool deserializeJson = true)`
Hope this is ok?